conformance/run_all.py

```
@@ -0,0 +1,276 @@
+    with tempfile.TemporaryFile() as out, tempfile.TemporaryFile() as err:
+        proc = subprocess.Popen(cmd, cwd=str(cwd), stdout=out, stderr=err)
+        deadline = time.monotonic() + timeout
+        try:
+            while True:
+                try:
+                    proc.wait(timeout=0.25)
+                    break
+                except subprocess.TimeoutExpired:
+                    pass
+                if out.tell() + err.tell() > OUTPUT_CAP_BYTES:
+                    proc.kill(); proc.wait()
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)